aws/table_aws_organizations_account.go

```diff
@@ -0,0 +1,173 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsOrganizationsAccount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_organizations_account",
+		Description: "AWS Organizations Account",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"AccountNotFoundException"}),
```

Comment:
Need to handle InvalidInputException.

> select * from aws_organizations_account where id = ''
Error: InvalidInputException: You provided a value that does not match the required pattern.

Reply:
Done